packages/http/src/shared.ts

```diff
@@ -201,3 +201,16 @@ export function getSignedTransactionFromActivity(
 
   return assertNonNull(`0x${signedTransaction}`);
 }
+
+/**
+ *
+ * A utility function to get a live timestmap directly from Turnkey
+ *
+ * @returns string reprenting timestamp in seconds
+ */
+export async function getLiveTimestamp(): Promise<string> {
+  const timestampResponse = await fetch("https://api.turnkey.com/health");
```

Comment:
I think we should make the hostname overridable or configurable? If I set my turnkey API host to preprod, this would also use preprod health.